cmd/config-migrate.go

```diff
@@ -88,33 +88,34 @@ func migrateConfig() error {
 
 // Version '1' is not supported anymore and deprecated, safe to delete.
 func purgeV1() error {
-	cv1, err := loadConfigV1()
-	if err != nil {
-		if os.IsNotExist(err) {
-			return nil
-		}
-		return fmt.Errorf("Unable to load config version ‘1’. %v", err)
-
-	}
+	configFile := filepath.Join(getConfigDir(), "fsUsers.json")
 
-	if cv1.Version == "1" {
-		// Purge old fsUsers.json file
-		configFile := filepath.Join(getConfigDir(), "fsUsers.json")
-		removeAll(configFile)
-		console.Println("Removed unsupported config version ‘1’.")
+	cv1 := &configV1{Version: "1"}
```

Comment:
Yes. Doesn't check and the check didn't work didn't work previously in `Load()`.